### ISSUE TITLE
Does not meet WCAG standards for accessible text

### ISSUE THREAD

**Issue by Korilakkuma** [WebAudio/web-audio-api#2257]
**Describe the Issue**

Does not meet WCAG standards for accessible text according to check by browser.
Actually, I think that  this contrast is hard to see ...

**Expected Result**

<img width="1440" alt="Expected Result" src="https://user-images.githubusercontent.com/4006693/95021747-dbb34580-06ad-11eb-8f3c-299006c07977.png">

**Actual Result**

<img width="1440" alt="Actual Result" src="https://user-images.githubusercontent.com/4006693/95021719-a9094d00-06ad-11eb-95ac-77c635f46e8c.png">

**Comment by rtoy** [comment]
Can you explain how you got the image for the actual result?  When I use Firefox (on mac), I pretty much see that the draft document looks very similar to the candidate recommendation image.  In particular, the table of contents is not white text on a black background; it's black text on a white background).  The "Latest published version" looks black to me. (The element inspector says the color is black: rgb(0,0,0).)

**Comment by Korilakkuma** [comment]
Maybe, the contrast is hard to see in the case of **dark mode** (macOS Catalina). Therefore, to set light mode, the contrast is easy to see.

**Comment by rtoy** [comment]
Ah, I see.  Yes, in dark mode, the draft spec is basically illegible.  But the latest published version looks fine.  I think we should update the draft to do whatever the published version does.  Would that work for you?

Or are you expecting the main text to work well in dark mode with white text on a black background?

**Comment by Korilakkuma** [comment]
> you expecting the main text to work well in dark mode with white text on a black background

Yes. Because the latest published version looks fine. Therefore, I expect that the main text to work well in dark mode with white text on a black background.

**Comment by rtoy** [comment]
Weird. On my Mac, the published version has black text on a white background when I turn on dark mode. Are you saying this should be white text on a black background?  That's not what I see in dark mode.

But I agree that the draft version is definitely broken in dark mode.  I'll have to dig in to the differences between the draft and published version. I don't think we made any changes about that.

**Comment by rtoy** [comment]
This could be related to bikeshed now supporting dark mode. See https://github.com/tabatkins/bikeshed/issues/1773

**Comment by rtoy** [comment]
See https://github.com/tabatkins/bikeshed/issues/1790.

**Comment by rtoy** [comment]
This should be fixed soon.  Thanks for reporting the issue.

**Comment by rtoy** [comment]
Bikeshed fix is done. I'll have to wait a bit until the changes propagate and then wait for the next PR to update the spec or force Travis to run and update the spec.

**Comment by Korilakkuma** [comment]
Thanks !

**Comment by rtoy** [comment]
By chance I found another typo in the spec. (Well, bikeshed-o).  When bikeshed is updated I'll fix the typo and get a new version up.  So I don't forget where it is:  https://webaudio.github.io/web-audio-api/#dom-baseaudiocontext-createbuffer-length.  It says "atleast 1", not "at least 1".  This is bug  tabatkins/bikeshed#1470.

**Comment by rtoy** [comment]
Our Travis CI is still using an older version of bikeshed without the fix.  Perhaps it's time to change our CI to clone the most recent copy of bikeshed instead of using pip3 install?

**Comment by rtoy** [comment]
@tabatkins said he'd update pip3 with a new release soon.  I'll check again soon.

**Comment by rtoy** [comment]
Bikeshed has been updated and I verified that the latest version has the fix.

**Comment by rtoy** [comment]
With #2262 fixed, the latest draft looks pretty good.  However, the [channel ordering table](https://webaudio.github.io/web-audio-api/#ChannelOrdering) is unreadable in dark mode.  We need  to fix that.

**Comment by rtoy** [comment]
The images and diagrams that we have look kind of garish (black text on a white background) when in dark mode.  But still quite legible.  I'm not keen on updating these at this time since they're still legible.  Something to keep in mind for v2 when we add diagrams.

**Comment by rtoy** [comment]
Also, we have custom colors for the tables for the [AudioNode info](https://webaudio.github.io/web-audio-api/#AudioBufferSourceNode) and [AudioParam info](https://webaudio.github.io/web-audio-api/#dom-audiobuffersourcenode-detune).  These seem ok in dark mode.  The contrast is a little low for the title, but not horrible.

**Comment by rtoy** [comment]
I think this is now all taken care of.  If you find any more issues with dark mode, please let us know and file new issues.
